mainwindow.cpp

```diff
@@ -278,19 +289,13 @@ MainWindow::~MainWindow()
 void MainWindow::on_pushButton_WordlistFileBrowse_clicked()
 {
     // We pop a dialog to choose a file to open.
-    // TODO: *.lst for file extension?
-    // TODO: Copy-pasting is evil! (open password file)
-    // TODO: What happens when John writes something while dialog
-    //       opened?
-    // TODO: Move dialog creation and setting up into window constructor.
     QFileDialog dialog;
     dialog.setFileMode(QFileDialog::ExistingFile);
-    // TODO: Dialog could allow user to select multiple files. May it
-    //       be good to support this ability? To concatenate selected file?
```

Comment:
On Thu, May 14, 2015 at 11:09:25PM -0700, Shinnok wrote:
> >      QFileDialog dialog;
> >      dialog.setFileMode(QFileDialog::ExistingFile);
> > -    // TODO: Dialog could allow user to select multiple files. May it
> > -    //       be good to support this ability? To concatenate selected file?
> 
> Aleksey, think it would be useful to have that, even after we get
> sessions working? If so then we can easily enable that. But no
> concatenation or file manipulation please.

I can't tell about real life but in contest we usually have several
files with hashes (and several files with hash format), so we use
commands like: john hashes/*.raw-md5.* or john --format=raw-md5
hashes/* , so the files are in 1 session. It is desirable for
non-salted hashes because additional hashes are attacked almost for
free.

No manipulations should occur. I agree.

Nevertheless it needs support from johnny because now it has only 1
file name: m_hashesFileName .

Thanks!

-- 
Regards,
Aleksey Cherepanov